x-pack/plugins/enterprise_search/public/applications/app_search/icons/cursor_icon.tsx

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
Adds a new icon. This directory will also include our icons for engines and meta_engines and all other app search specific icons we'll have. (Like on the relevance tuning pages)